ci_framework/roles/operator_build/tasks/build.yml

```diff
@@ -14,9 +14,16 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
+- name: "{{ operator.name }} - Get base module name from go.mod"
+  ansible.builtin.shell: |
+    grep {{ cifmw_operator_build_org }}/{{ operator.name }}/api go.mod | xargs basename
```

Comment:
`| xargs basename` is not needed as you'e again appending it in L26(?)

Reply:
Ok you have to split the version then.

Reply:
marking as resolved as we have changes the code